src/modules/Transcrippet/LineChart.tsx

```diff
@@ -0,0 +1,71 @@
+import { RealtimeTranscrippetPrice } from '@aiera/client-sdk/types/generated';
+import React, { RefObject } from 'react';
+
+interface LineChartProps {
+    data: RealtimeTranscrippetPrice[];
+    selectedIndex: number;
+    transcrippetRef: RefObject<HTMLDivElement>;
+}
+
+export const LineChart = ({ data, selectedIndex, transcrippetRef }: LineChartProps) => {
+    const containerWidth = transcrippetRef?.current?.getBoundingClientRect().width;
+    if (!containerWidth) return null;
+    const selectedItem = data[selectedIndex];
+
+    // Extracting price values from data
+    const prices = data.map((item) => item.price);
+
+    // Finding the highest and lowest prices
+    const minPrice = Math.min(...prices);
```

Comment:
Do you need to spread `prices` here or can you just do `Math.min(prices)` and `Match.max(prices)`?